source/includes/steps-install-shell-linux-deb.yaml

```diff
@@ -0,0 +1,31 @@
+---
+source:
+  file: steps-install-shell-base.yaml
+  ref: navigate-dlc
+ref: navigate-dlc-linux-deb
+---
+source:
+  file: steps-install-shell-base.yaml
+  ref: open-mdb-shell-page
+ref: open-mdb-shell-page-linux-deb
+---
+source:
+  file: steps-install-shell-base.yaml
+  ref: download-archive
+ref: download-archive-generic-linux
```

Comment:
• Step 1: You can direct users directly to tools DL page:

https://www.mongodb.com/try/download/tools

• Step 2: New DL center lists `mongosh` as "MongoDB Shell" not "Shell"

• Step 3: "DL for your platform" -- careful, Ubuntu is Linux as well. Since we have a dedicated tab for each, fair to just use what matches the tab:
- .tgz --> Download the "Linux 64-bit" ...
- .deb (Debian) --> Download the "Debian 64-bit" ...
- .rpm (RHEL)  --> "Download the 64-bit" ...

Possibly safe to skip the "64-bit", up to you.

• Step 3: However, this is rather hugely unfortunate, as the ".rpm (RHEL)" tab and "Rehat 64-bit" download will work for Amazon Linux and SUSE, likewise the ".deb (Debian)" tab and "Debian 64-bit" download will work for Ubuntu. Not sure how best to handle this. Certainly no need to bother with Ubuntu 20.04 vs Ubuntu 18.04 here, but I think adding something similar to "Use the .deb tab to install on the Debian and Ubuntu platforms" would be helpful.

Reply:
1. Done. Directed them right to shell download.
2. Blast, they changed it on us. Thanks for catching.
3. Yes, agreed this was a bit tricky. I like being explicit as you've suggested. Updating.
4. Added a clarifying section before the tabs.

**Question** Let me know if you would prefer the tab names to be the distributions (RHEL, Ubuntu/Debian, TGZ (ack that this is not a distribution)) rather than the package names. I went with package names since TGZ does not apply to a specific distro.

Reply:
I'd say you're absolutely right to go with package names over distro for exactly that reason. I think with the brief intro now preceding that second tier of tabs, we've added a lot of clarity to this user choice. Some thoughts about that intro inline.